packages/nextjs/src/index.types.ts

```diff
@@ -36,6 +36,8 @@ export declare const ErrorBoundary: typeof clientSdk.ErrorBoundary;
 export declare const showReportDialog: typeof clientSdk.showReportDialog;
 export declare const withErrorBoundary: typeof clientSdk.withErrorBoundary;
 
+export declare const Span: typeof edgeSdk.Span;
```

Comment:
h: Let's export the Span class from `@sentry/node` instead of adding this here.

Reply:
We're going to leave it as moving this will affect browser.